### PR TITLE
fix(Selection Controls): Dense Prop 

### LINKED ISSUE #9381: [Bug Report] Dense Prop for Selection Controls not working
### Environment
**Vuetify Version:** 2.1.4
**Vue Version:** 2.6.10
**Browsers:** Chrome 77.0.3865.90
**OS:** Mac OS 10.14.6

### Steps to reproduce
Adding `dense` prop to v-select and v-checkbox.

### Expected Behavior
Small icons/controls and tighter padding - reduced sizing in general

### Actual Behavior
Does not create any discernible difference between `dense` and non-dense

### Reproduction Link
<a href="https://codepen.io/Jasonrhaddix/pen/oNNLVBp" target="_blank">https://codepen.io/Jasonrhaddix/pen/oNNLVBp</a>

### PR DESCRIPTION
**Pull request by melbahja**
## Description
This PR require #9494 to work as expected

Works on:
- VSwitch
- VCheckbox
- VRadioGroup

## Motivation and Context
Fixes #9381 

## How Has This Been Tested?
visually | unit

## Markup:
```vue
 <v-switch
	dense
	label="Dense"
	color="primary"
></v-switch>


<v-switch
	dense
	inset
	label="Dense"
	color="primary"
></v-switch>

<v-checkbox
	dense
	label="Dense"
	color="primary"
></v-checkbox>


<v-radio-group dense>
	<v-radio label="Radio 1" value="radio-1"></v-radio>
	<v-radio label="Radio 2" value="radio-2"></v-radio>
</v-radio-group>
```

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

**Comment by melbahja** [comment]
@johnleider sorry about that, markup added.

File: packages/vuetify/src/components/VSwitch/_variables.sass [hunk 1]
```diff
@@ -14,3 +14,12 @@ $switch-thumb-height: 20px !default
 $switch-thumb-top: calc(50% - 10px) !default
 $switch-thumb-elevation: 4 !default
 $switch-dirty-offset-x: 20px !default
+
+
```
Comment (johnleider) [inline packages/vuetify/src/components/VSwitch/_variables.sass]:
```suggestion
```

File: packages/vuetify/src/components/VRadioGroup/VRadio.ts [hunk 2]
```diff
@@ -145,7 +145,12 @@ export default baseMixins.extend<options>().extend({
           ...this.attrs$,
         }),
         this.genRipple(this.setTextColor(this.validationState)),
-        this.$createElement(VIcon, this.setTextColor(this.validationState, {}), this.computedIcon),
+        this.$createElement(VIcon, this.setTextColor(this.validationState,
+          {
```
Comment (johnleider) [inline packages/vuetify/src/components/VRadioGroup/VRadio.ts]: Move this bracket onto line 48

**Comment by DRoet** [comment]
Thanks for all your work on the `dense` stuff ❤️

**Comment by melbahja** [comment]
@DRoet :smile:

**Pull request merged**